server/handler.go

```diff
@@ -21,25 +21,28 @@ import (
 
 var regKillCmd = regexp.MustCompile(`^kill (?:(query|connection) )?(\d+)$`)
 
-var errConnectionNotFound = errors.NewKind("Connection not found: %c")
+var errConnectionNotFound = errors.NewKind("connection not found: %c")
+var ErrRowTimeout = errors.NewKind("row read wait bigger than connection timeout")
```

Comment:
minor nit: This should have a doc comment, or be unexported.

Reply:
Done.